tests/test_Consumer.py

```diff
@@ -41,6 +41,17 @@ def dummy_assign_revoke(consumer, partitions):
     if msg is not None:
         assert msg.timestamp() == (TIMESTAMP_NOT_AVAILABLE, -1)
 
+    msglist = kc.consume(num_messages=10, timeout=0.001)
+    if len(msglist) == 0:
```

Comment:
aren't we expecting exactly 0 messages here? use `assert len(msglist) == 0, "expected 0 messages, not %d" % len(msglist)`